apps/server/src/modules/user-import/uc/user-import.uc.spec.ts

```diff
@@ -114,7 +114,7 @@ describe('[ImportUserModule]', () => {
 			});
 		};
 
-		const createMockSchoolDo = (school?: School): SchoolDO => {
+		const createMockSchoolDo = (school?: School): LegacySchoolDo => {
```

Comment:
An entity as a parameter is expected, but in the test case "Should save school params" (line 637), the parameter in the school service is a domain object. However, I am currently not sure why this error is manifesting in my code and not in the main part.